RuddockWebsite/validation_utils.py

```diff
@@ -0,0 +1,124 @@
+import re
+import datetime
+from flask import flash, g
+from sqlalchemy import text
+from RuddockWebsite import constants
+
+def validate_username(username, flash_errors=True):
+  '''
+  Checks to make sure username is valid. If flash_errors is True, then the
+  errors will be displayed as flashes. Returns True if valid, else False.
+  '''
+  error = None
+  username_regex = re.compile(r'^[a-zA-Z0-9\-\_]+$')
```

Comment:
you should make this global within the module so it doesn't have to be recompiled every time this function is called, which makes it less useful